### PR TITLE
[WB-7682] Add yea tests checking logged parameter and gradient names

### PR DESCRIPTION
**Pull request by KyleGoyette**
[Fixes WB-7682](https://wandb.atlassian.net/browse/WB-7682)

Description
-----------
Adds yea tests for tf2.5 and tf2.6 to check logged parameter and gradient names for keras.

Testing
-------
This is test

Checklist
-------
- Name PR "[WB-NNNN][WB-MMMM] Add support for..." similar to entries in CHANGELOG.md
- Include reference to internal ticket "Fixes WB-NNNN" (and github issue "Fixes #NNNN" if applicable)

### CODE REVIEW

**Review by raubitsj — APPROVED** [review]
Great.
